edx_notifications/channels/urban_airship.py

```diff
@@ -117,10 +117,32 @@ def bulk_dispatch_notification(self, user_ids, msg, exclude_user_ids=None, chann
                 if user_id not in exclude_user_ids:
                     actual_user_ids.append(user_id)
             payload = self.create_bulk_user_payload(actual_user_ids, msg)
+
+        self._add_type_in_payload(msg, payload)
         payload = json.dumps(payload)
         api_credentials = channel_context.get('api_credentials') if channel_context else None
         return self.call_ua_push_api(payload, api_credentials)
 
+    def _add_type_in_payload(self, msg, payload):
+        """
+        Adds a notification type in payload if notification_type is coming in the message
+        :param msg:
+        :param payload:
+        :return:
```

Comment:
nit: if this method does not return anything we should remove `return` in docstring.